components/bl-mapbox-component/src/actions.js

```diff
@@ -0,0 +1,21 @@
+export function createActions(mapRef, component) {
+  const map = mapRef.current;
+
+  Object.assign(component, {
+    isMoving  : () => map?.isMoving(),
```

Comment:
If there are no map at this moment, your assignation doesn't make anything.
So I think we should avoid id just added condition

```
if (map) {
 ...
}
```